apps/src/code-studio/components/stageExtras/BonusLevels.jsx

```diff
@@ -1,19 +1,23 @@
+import FontAwesome from '../../../templates/FontAwesome';
```

Comment:
Tiny nit: can also do `@cdo/apps/templates/FontAwesome` here (will preserve reference even if this file is moved).